@navikt/internal/css/header.css

```diff
@@ -0,0 +1,66 @@
+:root {
+  --navdsi-header-color-background: var(--navds-color-darkgray);
```

Comment:
Klarer vi å finne en bedre måte å prefikse navnet på?
core: navds
internal: navdsi
navno: navds??

Reply:
Jeg liker egentlig ikke navnet `navdsi`, men måtte bare ha noe for å komme videre. Har du noen ideer?